kgforge/specializations/models/rdf/store_service.py

```diff
@@ -68,93 +71,95 @@ def resolve_context(self, iri: str) -> Dict:
         return document
 
     def generate_context(self) -> Dict:
-        for v in self._shapes_to_resources.values():
-            self._load_shape(v)
+        for shape_uriref, schema_uriref in self.shape_to_defining_resource.items():
+            if str(schema_uriref) not in self._imported:
+                self.transitive_load_shape_graph(
+                    self._get_named_graph_from_shape(shape_uriref), schema_uriref
+                )
         # reloads the shapes graph
         self._sg = ShapesGraphWrapper(self._graph)
```

Comment:
This can be changed to self._init_shape_graph_wrapper()

Reply:
There's an unused import resulting from that (ShapesGraphWrapper doesn't need to be imported)